driver-reactive-streams/src/main/com/mongodb/reactivestreams/client/internal/BatchCursorPublisher.java

```diff
@@ -122,10 +122,8 @@ public void subscribe(final Subscriber<? super T> subscriber) {
                     if (batchSize != null) {
                         batchCursor.setBatchSize(batchSize);
                     }
-                    return Flux.create((FluxSink<T> sink) -> {
-                        sink.onRequest(value -> recurseCursor(sink, batchCursor, inProgress));
-                        recurseCursor(sink, batchCursor, inProgress);
```

Comment:
This call just races against the sink.onRequest call - so removed.

Reply:
Ack.